packages/utils/src/normalize.ts

```diff
@@ -214,6 +214,10 @@ function stringifyValue(
       return '[Document]';
     }
 
+    if (isVueViewModel(value)) {
+      return '[VueViewModel]';
+    }
+
     // React's SyntheticEvent thingy
     if (isSyntheticEvent(value)) {
       return '[SyntheticEvent]';
```

Comment:
well I guess there's precedence for handling this stuff like this in a general package of ours 😅